README.md

```diff
@@ -26,15 +26,15 @@ information, consult the [hex docs](https://hexdocs.pm/nerves_runtime).
 initialize the system when it is started. For this to be useful,
 `nerves_runtime` must be started before other OTP applications, since most will
 assume that the system is already initialized before they start. To set up
-`nerves_runtime` to work with `bootloader`, you will need to do the following:
+`nerves_runtime` to work with `shoehorn`, you will need to do the following:
 
-1.  Include `bootloader` in `mix.exs`
-2.  Include `Bootloader.Plugin` in your `rel/config.exs`
+1.  Include `shoehorn` in `mix.exs`
+2.  Include `shoehorn` in your `rel/config.exs`
```

Comment:
Should this be `Shoehorn` instead of `shoehorn`? I looked around a little and didn't see an example, even in the `shoehorn` readme, about what to put where for the `rel/config.exs` file.

Reply:
Yup. Not near a term though